pkgs/repo_manage/lib/issue_transfer.dart

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:github/github.dart';
+import 'package:graphql/client.dart';
+
+import 'src/common.dart';
+
+class TransferIssuesCommand extends ReportCommand {
+  TransferIssuesCommand()
+      : super('transfer-issues',
+            'Bulk transfer issues from one repo to another.') {
+    argParser.addFlag(
+      'apply-changes',
+      negatable: false,
+      help: 'WARNING: This will transfer the issues. Please preview the changes'
+          "first by running without '--apply-changes'.",
+      defaultsTo: false,
+    );
+    argParser.addMultiOption(
+      'issue-numbers',
```

Comment:
'`issues'`? Perhaps that's not clear enough that we're expecting numbers instead of something like links.